cluster/gce/gci/configure-helper.sh

```diff
@@ -1593,4 +1593,5 @@ else
 fi
 reset-motd
 prepare-mounter-rootfs
+modprobe configs
```

Comment:
Oh, Nvidia driver installation may not require configs on all distros. I'm not sure about that.